index.html

```diff
@@ -174,9 +178,9 @@ <h1 class="about-title">About Myself</h1>
                     <li class="tab-item">
                         <div class="tab-title"><span class="tab-text">Languages</span> <i class="fa-solid fa-arrow-down"></i></div>
                         <ul class="tab-langs">
-                            <li class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1.png" alt="javascript"><span class="tab-text">JavaScript</span></li>
-                            <li class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1(1).png" alt="html"><span class="tab-text">HTML</span></li>
-                            <li class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1(2).png" alt="css"><span class="tab-text">CSS</span></li>
+                            <div class="tab-lang"><img src="Microverse-Student-Project-1/Ellipse 1.png"><span class="tab-text">JavaScript</span></div>
```

Comment:
- [ ] I noticed you removed all `alt` attributes for the images as you were implementing the changes suggested by the previous reviewer. Kindly return them. Using `alt` attributes when using the `img` tags makes it easy for screen readers to capture images for people who use screen readers to navigate web applications. 👍